lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -112,6 +126,14 @@ def custom_transform
         def ignored_words
           cop_config.fetch('IgnoredWords', [])
         end
+
+        def insufficient_examples
+          cop_config.fetch('DisallowedExamples', [])
+        end
+
+        def clean_msg(message)
+          message.strip.squeeze(' ').downcase
```

Comment:
If I format it with `downcase`, won't it always mismatch if there are uppercase letters in the `DisallowedExample` setting?

Reply:
good callout! we should not allow uppercase in the DisallowedExample setting! Updating!

Reply:
great callout. I've added a comment that the words in this config will not be treated as case sensitive!

Reply:
I think it looks good.
Also, would it look good to add the following test?

```ruby
context 'when `DisallowedExamples: Workz`' do
  let(:cop_config) { { 'DisallowedExamples' => ['Workz'] } }

  it 'flags an unclear description' do
    expect_offense(<<-'RUBY')
      it "workz" \
          ^^^^^^^^ Your test should be more descriptive.
        "with #{object}" do
      end
    RUBY
  end

  it 'flags an unclear description despite extra spaces' do
    expect_offense(<<-'RUBY')
      it "  workz    " \
          ^^^^^^^^^^^^^^ Your test should be more descriptive.
        "with #{object}" do
      end
    RUBY
  end

  it 'flags an unclear description despite uppercase and lowercase strings' do
    expect_offense(<<-'RUBY')
      it "WOrKz " \
          ^^^^^^^^^ Your test should be more descriptive.
        "with #{object}" do
      end
    RUBY
  end
end
```

Reply:
@ydah  I thought I saw a notification on updating the test cases! that was a good idea! have added that too! 
Appreciate your patience on this! 
PR is ready for review again 😅

Reply:
Oops, I thought I had added a comment about the proposed test, but it disappeared. Thank you so much!